src/content/plugins/define-plugin.md

```diff
@@ -101,3 +101,15 @@ new webpack.DefinePlugin({
   'SERVICE_URL': JSON.stringify('https://dev.example.com')
 });
 ```
+
+### Rebuild 
+
+Rebuild wehen the schema updates.
```

Comment:
```suggestion
Rebuild when the schema updates.
```

Reply:
I think Schema was only mentioned in the PR as an example... This feature allows you to actually watch for changes in runtimeValue or am i missing something? https://github.com/webpack/webpack/pull/6793/files

Reply:
I think you are correct